waitress/tests/test_parser.py

```diff
@@ -394,6 +418,15 @@ def test_parse_header_multiple_values_header_folded_multiple(self):
         self.assertIn("FOO", self.parser.headers)
         self.assertEqual(self.parser.headers["FOO"], "bar, whatever, more, please, yes")
 
+    def test_parse_header_multiple_values_extra_space(self):
+        # Tests errata from: https://www.rfc-editor.org/errata_search.php?rfc=7230&eid=4
```

Comment:
this link doesn't take me anywhere useful

Reply:
I guess you meant `https://www.rfc-editor.org/errata_search.php?rfc=7230&eid=4189` from other spots in the code

Reply:
Fixed. 👍